yarn.lock

```diff
@@ -1942,6 +1942,15 @@
     "@griffel/react" "^1.5.22"
     "@swc/helpers" "^0.5.1"
 
+"@fluentui/react-charts@*":
```

Comment:
this change is incorrect. We have not taken any new dependencies. There should be not change in yarn.lock